### ISSUE TITLE
Add testID support

### ISSUE THREAD

**Issue by sbalay** [kristerkari/react-native-svg-transformer#127]
This is a feature request, it would be nice to allow sending `testID` to components generated when importing `.svg` files. `testID` is a standard prop used by several testing frameworks like @testing-library/react-native or detox.

This prop is already available when using SVG from react-native-svg:

```jsx
import SVG from 'react-native-svg'

function MySVG({children}) {
  return <SVG testID="myTestID">{children}</SVG>
}
```

so it would be nice to be able to do the same in the following scenario

```jsx
import Logo from 'logo.svg'

function MyLogo() {
  // currently testID is not supported as a prop
  return <Logo testID="myTestID" />
}
```

> Disclaimer: This issue should probably be created in svgr instead, but I'm not sure. Let me know if that's the case and I'll close this one.

**Comment by kristerkari** [comment]
Thanks @sbalay ! I don't know if `SVGR` or `react-native-svg` have support for the `testID` property or not. To find that out, you could test using just plain `react-native-svg` components and see if the `testID` property works there.

**Comment by sbalay** [comment]
> Thanks @sbalay ! I don't know if `SVGR` or `react-native-svg` have support for the `testID` property or not. To find that out, you could test using just plain `react-native-svg` components and see if the `testID` property works there.

Thanks @kristerkari! testID works fine when using the SVG component from `react-native-svg`:

```
import SVG from 'react-native-svg'

function MySVG({children}) {
  return <SVG testID="myTestID">{children}</SVG>
}
```

Not sure if that's what you meant though

**Comment by kristerkari** [comment]
> Not sure if that's what you meant though

Yes, that's what I meant! I think that the problem might be in SVGR then.
